can/interfaces/usb2can/serial_selector.py

```diff
@@ -27,14 +29,19 @@ def WMIDateStringToDate(dtmDate):
     return strDateTime
 
 
-def serial():
-    strComputer = '.'
+def find_serial_devices(serial_matcher="ED"):
+    """
+    Finds a list of serial devices.
```

Comment:
Finds a list of USB devices where the serial number (partially) matches the given string.